migrations/app/20171010160121_app_workload_points_t2a_column.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex, promise) {
+  var sql = 'ALTER TABLE app.workload_points ADD isT2A BIT;'
```

Comment:
in order to keep consistent with the rest of our fields this should be in snake case i.e. `is_t2a`.

Reply:
👍